### DISCUSSION TITLE
Ne pas générer les enfants

### DISCUSSION THREAD

**Discussion by sebfar9172** [MadeInPierre/finalynx#108] [Q&A] [answered]
Hello
Par exemple dans la section crowdlending je n'ai pas besoin d'avoir les lignes de chaque prêt mais uniquement le montant global. Est ce faisable ?

**Comment by MadeInPierre** [comment, marked as answer]
Juste au cas où quelqu'un tombe sur cette discussion plus tard, oui on peut ^^

```python
Folder(..., display=FolderDisplay.COLLAPSED) # or LINE
```

**Reply by sebfar9172** [reply]
Quelle est la différence entre collapsed et Line ?

**Reply by MadeInPierre** [reply]
De mémoire ça ne change rien pour le web, par contre dans le terminal :
- EXPANDED affichage normal, dossier en bleu gras et enfants en dessous 
- COLLAPSED les enfants ne sont plus affichés, dossier en bleu non gras
- LINE enfants non affichés, dossier noir non gras (comme une ligne)

Par exemple mes AV pilotées sont en LINE, mais mes dossiers sont en COLLAPSED quand j'ai envie de réduire la taille de l'arbre ^^
